### PR TITLE
liuliu/return databasename

### PR DESCRIPTION
**Pull request by liuliu-dev**

### CODE REVIEW

File: go/libraries/doltcore/doltdb/doltdb.go [hunk 1]
```diff
@@ -97,6 +97,10 @@ func DoltDBFromCS(cs chunks.ChunkStore, databaseName string) *DoltDB {
 	return &DoltDB{db: hooksDatabase{Database: db}, vrw: vrw, ns: ns, databaseName: databaseName}
 }
 
+func (ddb *DoltDB) GetDatabaseName() string {
```
Comment (zachmu) [inline go/libraries/doltcore/doltdb/doltdb.go:100]: Add a method doc with a note that this can be empty in some cases

**Review by zachmu — APPROVED** [review]
LGTM

**Comment by coffeegoddd** [comment]
@liuliu-dev __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>c45bf84</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>c45bf84</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Comment by coffeegoddd** [comment]
@liuliu-dev __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>cc3c252</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>cc3c252</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Pull request merged**